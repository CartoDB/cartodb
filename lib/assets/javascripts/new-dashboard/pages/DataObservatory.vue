<template>
  <Page class="page--data">
    <DataSecondaryNavigation></DataSecondaryNavigation>
    <section class="section section--noBorder">
      <DataObservatoryList></DataObservatoryList>
    </section>
  </Page>
</template>

<script>
import Page from 'new-dashboard/components/Page';
import DataSecondaryNavigation from 'new-dashboard/components/NavigationBar/DataSecondaryNavigation.vue';
import DataObservatoryList from 'new-dashboard/components/DataObservatoryList.vue';

export default {
  name: 'DataObservatoryPage',
  components: {
    DataSecondaryNavigation,
    Page,
    DataObservatoryList
  }
};
</script>
<style scoped lang="scss">
@import 'new-dashboard/styles/variables';

.page.page--data {
  padding: 64px 0 0;
}

.subnavbar-elementItem {
  margin-left: 28px;
  text-decoration: none;

  &.is-active {
    position: relative;
    color: #2E3C43;

    &::after {
      content: "";
      position: absolute;
      bottom: -24px;
      left: 0;
      width: 100%;
      height: 4px;
      background-color: currentColor;
    }
  }
}
</style>
