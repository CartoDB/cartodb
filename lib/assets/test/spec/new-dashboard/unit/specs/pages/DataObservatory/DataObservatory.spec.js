import { shallowMount } from '@vue/test-utils';
import DataObservatory from 'new-dashboard/pages/DataObservatory';

describe('DataObservatory.vue', () => {
  describe('Rendering', () => {
    it('should render correctly', () => {
      const dataobservatoryPage = shallowMount(DataObservatory);
      expect(dataobservatoryPage).toMatchSnapshot();
    });
  });
});
