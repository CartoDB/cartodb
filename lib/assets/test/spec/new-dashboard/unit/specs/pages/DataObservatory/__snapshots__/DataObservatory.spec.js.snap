// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataObservatory.vue Rendering should render correctly 1`] = `
<page-stub class="page--data">
  <datasecondarynavigation-stub></datasecondarynavigation-stub>
  <section class="section section--noBorder">
    <dataobservatorylist-stub></dataobservatorylist-stub>
  </section>
</page-stub>
`;
